cypress/fixtures/upload/file.txt

```diff
@@ -0,0 +1 @@
+dit is tekst 1
```

Comment:
Wat is dit?

Reply:
Dit wordt gebruikt in `vl-upload-next` component test.

Meer specifiek in deze lijn:

`const txtFileFixturePath = 'cypress/fixtures/upload/file.txt';`